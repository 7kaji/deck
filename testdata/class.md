# Inline Style

---

# Title

## Hello style

- Hello <span class="green" >green</span>
- <span  class='blue'>blue</span> world

<span class=" red">red</span> world
