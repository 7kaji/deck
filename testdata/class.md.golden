[
  {
    "layout": "",
    "titles": [
      "Inline Style"
    ]
  },
  {
    "layout": "",
    "titles": [
      "Title"
    ],
    "subtitles": [
      "Hello style"
    ],
    "bodies": [
      {
        "paragraphs": [
          {
            "fragments": [
              {
                "value": "Hello "
              },
              {
                "value": "green",
                "className": "green"
              }
            ],
            "bullet": "-"
          },
          {
            "fragments": [
              {
                "value": "blue",
                "className": "blue"
              },
              {
                "value": " world"
              }
            ],
            "bullet": "-"
          },
          {
            "fragments": [
              {
                "value": "red",
                "className": "red"
              },
              {
                "value": " world"
              }
            ]
          }
        ]
      }
    ]
  }
]